enclave_build/src/storage.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+#![allow(dead_code)]
+use std::{
+    fs::{self},
+    path::{Path, PathBuf},
+};
+
+use oci_spec::image::ImageConfiguration;
+
+use crate::{EnclaveBuildError, Result};
+
+/// Root folder for the image storage.
+pub const STORAGE_ROOT_FOLDER: &str = "XDG_DATA_HOME";
```

Comment:
Do we need these as pub?

Reply:
Might be used in the future as public, will remove it for now.